_posts/blog/2018-11-12/2018-11-12-how-will-the-bch-hard-fork-on-nov-15-affect-openbazaar.markdown

```diff
@@ -0,0 +1,33 @@
+---
+title: "How Will The <span class='post-title-extra'>BCH Hard Fork</span> Affect OpenBazaar?"
+layout: post
+date: '2018-11-12 06:30:00 -0300'
+social_title: 'How Will The Bitcoin Cash (BCH) Hard Fork on November 15th Affect OpenBazaar?'
+social_description: 'Here is what you need to know about the Bitcoin Cash hard fork on November 15th as it relates to Openbazaar.'
+---
+
+#### What’s Happening with Bitcoin Cash?
+
+There is a [Bitcoin Cash](https://www.bitcoincash.org/) network upgrade due to complete at 4:40PM UTC on November 15th, 2018. Depending on how the upgrade is accepted by the network, the Bitcoin Cash blockchain could fork into two or more incompatible chains. In order to help OpenBazaar users prepare for the fork, here is some information about the OpenBazaar wallet that can help you decide how to handle any BCH funds that you may have stored there.
+
+#### How Will The Bitcoin Cash Upgrade Affect My OpenBazaar Funds?
+
+Users of OpenBazaar 2.2.5 and prior who have an active BCH wallet are using a local SPV-based wallet. [SPV wallets](https://bitcoin.org/en/glossary/simplified-payment-verification) are designed to follow [Nakamoto Consensus](https://blockonomi.com/nakamoto-consensus/), which means the fork that has the most cumulative proof-of-work according to the existing Bitcoin Cash difficulty rules. 
```

Comment:
This statement about Nakamoto Consensus is incorrect and misleading. SPV wallets do not follow NC, they are designed to follow the chain the most proof of work that is compatible with the same block header and difficulty rules. The end of the statement is correct.

I'd say something like
> SPV wallets are designed to the chain that has the most cumulative proof-of-work according to the existing Bitcoin Cash difficulty rules.

Reply:
Updated